src/redux/actionCreators.js

```diff
@@ -191,22 +191,59 @@ export function saveGoogleDrive(file) {
   };
 }
 
-export function openGoogleDrive(file) {
+export function openGoogleDrive() {
   return dispatch => {
     dispatch({type: actType.START_OPEN_DRIVE});
-    var promise = new Promise((resolve, reject) => {
-      window.setTimeout(() => resolve(file), 1000);
-      if (!file) {
-        reject(new Error("File not opened"));
+    var developerKey = 'AIzaSyBY8GEXHLa5qYjQikWOej-N3DsckD3zAm4';
+    var clientId = '57458865722-8n3hcg08ogd39na132lhpdrc5on7lj29.apps.googleusercontent.com';
```

Comment:
Can you stick these in the .env file the same way we did with the firebase config?